bin/check-rabbitmq-queues-synchronised.rb

```diff
@@ -32,6 +32,12 @@ class CheckRabbitMQQueuesSynchronised < Sensu::Plugin::RabbitMQ::Check
          boolean: true,
          default: false
 
+  option :verify_ssl_off,
```

Comment:
I think I'd prefer either just calling it `verify_ssl` or `insecure` (matching `curl`) as it avoid the mental gymnastics of double negation. If you choose to use `verfy_ssl` you should change the default to true as we should have secure defaults and keep backwards compatibility. That being said that's how the other checks are named so we should at least be consistent with out collective madness I guess.